app/views/support/ABTest.scala

```diff
@@ -139,6 +140,25 @@ object MessageCopyTest extends TestTrait {
   )
 }
 
+object PaymentMethodTest extends TestTrait {
+
+  def name = "PaymentMethodTest"
+
+  def slug = "paymentMethods"
+
+  case class PaymentMethodVariantData(paymentMethods: Set[String]) extends VariantData
+
+  object PaymentMethodVariantData {
+    implicit val format: Format[PaymentMethodVariantData] = Json.format[PaymentMethodVariantData]
+  }
+
+  def variants = NonEmptyList(
+    makeVariant("Control", "control", 1, PaymentMethodVariantData(Set("CARD"))),
```

Comment:
paymentMethods could be a trait with Card and PayPal as case objects to give compile time safety instead of using Strings.